src/filter-builder.js

```diff
@@ -1,46 +1,19 @@
-import _ from 'lodash'
-import { boolMerge, buildClause } from './utils'
-import queryBuilder from './query-builder'
-import aggregationBuilder from './aggregation-builder'
+import { pushQuery, toBool } from './utils'
 
-export default function filterBuilder () {
-  let filter = {}
-
-  function addMinimumShouldMatch(str) {
-    const shouldClause = _.get(filter, 'bool.should')
-    if (shouldClause && shouldClause.length > 1) {
-      filter.bool['minimum_should_match'] = str
-    }
+export default function filterBuilder (options) {
+  const filters = {
+    and: [],
+    or: [],
+    not: []
```

Comment:
Another optional comment: I think you could simplify the `toBool` logic that comes later by naming these `must`, `should` and `must_not` right here.